index.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html>
+    
+ 
+<head></head> 
+    <link rel="stylesheet" href="styles.css">
+    
+
```

Comment:
Kindly add the styles.css link inside the head tag <head>. It  is not advisable to leave the css link outside of the head tag because it is considered not to be semantic and it is html coding practice.